### ISSUE TITLE
Change video filename below player controls!!!

### ISSUE THREAD

**Issue by nikos-ioannidis** [yixia/VitamioBundle#201]
Can i change the filename that is showing below video controls?

**Comment by djahmednawaz** [comment]
Yes. setFileName to your mediaController object on onPrepared method like following.

mediaController = new MediaController(this);
        videoView.setMediaController(mediaController);
        videoView.setMediaBufferingIndicator(mBufferingIndicator);
        videoView.requestFocus();
        videoView.setOnPreparedListener(new MediaPlayer.OnPreparedListener() {
            @Override
            public void onPrepared(MediaPlayer mediaPlayer) {
                mediaPlayer.setPlaybackSpeed(1.0f);
                mediaController.setFileName(title);
            }
        });

**Comment by MelissaTaylor** [comment]
@djahmednawaz  @crossle Can I add an icon to SetFileName?
